src/sheet_metal_conus.py

```diff
@@ -3,188 +3,338 @@
 # Distributed under the terms of the GNU Lesser General Public License v3.0
 
 import sys
-sys.path.append('/usr/share/inkscape/extensions')
+#sys.path.append('/usr/share/inkscape/extensions')
```

Comment:
It worked with my installation - hope it will be ok with installations on different patforms, too.